src/lib/seed.ts

```diff
@@ -23,15 +23,16 @@ export class InvalidSeed {
   }
 }
 
-export const defaultSeedPort = 8777;
-export const defaultNodePort = 8776;
+export const defaultSeedPort = window.HEARTWOOD ? 8080 : 8777;
+export const defaultNodePort = window.HEARTWOOD ? 9050 : 8776;
```

Comment:
`defaultNodePort` is the same as before, no need to check for heartwood

Reply:
Was the `9050` a typo in your original PR?

Reply:
yes

Reply:
I mentioned it somewhere, but can't find it

Reply:
506b942e1875a30187d27aa08ff9ff75f4433cc9